cms/djangoapps/contentstore/management/commands/compare_course_index_entries.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from xmodule.modulestore.split_mongo.mongo_connection import MongoPersistenceBackend, DjangoFlexPersistenceBackend
+from xmodule.modulestore.django import modulestore
+from django.core.management.base import BaseCommand
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+
+    """
+    A Command to determine if the Mongo active_versions and Django course_index tables are out of sync with one another.
+    """
+
+    def handle(self, *args, **options):
+        module_store = modulestore()
+        courses = module_store.get_courses()
+        course_ids = [x.id for x in courses]
+        out_of_sync = {}
+
+        mongostore = MongoPersistenceBackend(**settings.CONTENTSTORE['DOC_STORE_CONFIG'])
+        msqlstore = DjangoFlexPersistenceBackend(**settings.CONTENTSTORE['DOC_STORE_CONFIG'])
+
+        def compare_entries(mongo_entry, msql_entry, course_id):
+            draft = False
+            published = False
+            if(mongo_entry["draft-branch"] != msql_entry["draft-branch"]):
+                draft = True
+            if(mongo_entry["published-branch"] != msql_entry["published-branch"]):
+                published = True
+            if (draft | published):
+                out_of_sync[course_id._to_string()] = (draft, published)
```

Comment:
You can call `str(course_id)` here.

Also, you'll probably want more verbose output of what those values are, instead of just saying they're different.